libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm

```diff
@@ -13,3 +13,7 @@ SERIAL5_PROTOCOL 1
 # lower heater gains
 BRD_IMUHEAT_I 0.07
 BRD_IMUHEAT_P 50
+
+# compass primary defaults
+COMPASS_PRIO1_ID 73475
+COMPASS_PRIO2_ID 592905
```

Comment:
I'd also suggest setting COMPASS_PRIO3_ID to 0, to ensure it doesn't have a conflict

Reply:
Done. Thanks!